packages/react/src/components/Link/Link.module.css

```
@@ -0,0 +1,28 @@
+.link,
+.link:visited {
+  color: var(--fds-semantic-text-action-primary-default);
+}
+
+.link:hover {
+  color: var(--fds-semantic-text-action-primary-hover);
+  text-decoration: none;
+}
+
+.link:active,
+.link:focus {
+  background: var(--fds-semantic-border-focus-outline);
+  border-bottom: 2px solid var(--fds-semantic-border-focus-boxshadow);
```

Suggest changing this to box-shadow as we go from no border to border, which can shift content bellow.

```suggestion
  box-shadow: 0 2px 0 var(--fds-semantic-border-focus-boxshadow);
```